src/test/java/com/basho/riak/client/core/util/DefaultCharsetTest.java

```diff
@@ -31,7 +34,7 @@ public void tearDown()
     }
 
     @Test
-    public void testSystemDefaultCharsetIsReturnedIfNoPropertyIsDefined()
+    public void a_testSystemDefaultCharsetIsReturnedIfNoPropertyIsDefined()
```

Comment:
They are, but I realized this smells more than I wanted. going to massage these tests a little more :/